setup.py

```diff
@@ -46,6 +46,8 @@
         "fairseq2n" + fairseq2n_version_spec,
         "jiwer~=3.0",
         "numpy~=1.23",
+        "pyarrow>=13.0.0",
+        "pandas>=2.0.0",
```

Comment:
to avoid this extra deps that could be heavy, we could provide specific installation with `pip install fairseq2[arrow]`.